### PR TITLE
Update to version 0.2.3

### PR DESCRIPTION
**Pull request by phobson**

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -21,7 +20,7 @@ requirements:
   run:
     - python
     - numpy
-    - matplotlib <2
+    - matplotlib
```
Comment (ocefpaf) [inline recipe/meta.yaml]: :+1:

File: recipe/meta.yaml [hunk 2]
```diff
@@ -33,10 +32,10 @@ test:
     - pytest-mpl
     - scipy
   commands:
-    - python -c "import sys; import matplotlib as mpl; mpl.use('agg'); import probscale as ps; sys.exit(ps.test('--mpl'))"
+    - python -c "import sys; import matplotlib as mpl; mpl.use('agg'); import probscale as ps; sys.exit(ps.test())"
 
 about:
-  home: http://phobson.github.io/mpl-probscale/
+  home: http://matplotlib.org/mpl-probscale/
```
Comment (ocefpaf) [inline recipe/meta.yaml]: Nice!

**Comment by phobson** [comment]
@ocefpaf Any thoughts on what I did to make Travis so mad? Should I cancel and restart the build?

**Comment by ocefpaf** [comment]
> @ocefpaf Any thoughts on what I did to make Travis so mad? Should I cancel and restart the build?

It is not you. Travis-CI is awfully slow these days and we need to wait for it to process the backlog. But it seems to be green now, so let's merge this!

**Pull request merged**